apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -14,14 +14,39 @@ const LinkHomepageCodec = t.strict({
   ]),
 });
 
+const MediaCodec = t.strict({
+  attributes: t.strict({
+    name: t.string,
+    width: t.number,
+    height: t.number,
+    ext: t.string,
+    mime: t.string,
+    url: t.string,
+  }),
+});
+
+const ProductCodec = t.strict({
+  attributes: t.strict({
+    name: t.string,
+    description: t.string,
+    slug: t.string,
+    logo: t.strict({ data: MediaCodec }),
```

Comment:
```suggestion
    // TODO: remove null when Strapi will manage media in localhost properly
    logo: t.strict({ data: t.union([t.null, MediaCodec]) }),
```
I added [this task](https://pagopa.atlassian.net/jira/software/projects/DEV/boards/280/backlog?epics=visible&selectedIssue=DEV-1401) to resolve this TODO .